components/collective-page/hero/HeroAvatar.js

```diff
@@ -101,14 +120,12 @@ const HeroAvatar = ({ collective, isAdmin, intl }) => {
           disabled={submitting}
           inputProps={{ style: { width: 1 } }}
           onDrop={acceptedFiles => {
-            setUploadedImage(
-              ...acceptedFiles.map(file =>
-                Object.assign(file, {
-                  preview: URL.createObjectURL(file),
-                }),
-              ),
-            );
-            setEditing(true);
+            const image = acceptedFiles.map(file => Object.assign(file, { preview: URL.createObjectURL(file) }))[0];
+            if (image) {
+              validateImage(image);
+              setUploadedImage(image);
```

Comment:
Does it make sense to add the image to the state if it's not valid anyway? Maybe you could use validateImage inside the "if" (return true if valid or false if invalid).